packages/components/src/Headline/Headline.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
why is this needed here and not on other components? And is this not solved through `index.ts` in `docs/ui`?

Reply:
I think this can be removed

Reply:
ah no cant because of context ussage inside `react-aria` :D

Reply:
hm okay